app/lib/efile/nj/nj1040_calculator.rb

```diff
@@ -477,7 +477,8 @@ def calculate_line_56
       end
 
       def calculate_line_57
-        @intake.estimated_tax_payments&.round
+        return nil if @intake.estimated_tax_payments.nil? && @intake.overpayments.nil?
+        ((@intake.estimated_tax_payments || 0) + (@intake.overpayments || 0)).round
```

Comment:
[pebble] if they answered "yes" to `has_estimated_payments`, they're required to at least type in 0 for both these fields, right? so there shouldn't be a case where only one of them is `nil`. might be clearer to actually reference `has_estimated_payments` in this method, like this:
```
if @intake.has_estimated_payments
  (@intake.estimated_tax_payments + @intake.overpayments).round
end
```

Reply:
actually, per [this comment](https://github.com/codeforamerica/vita-min/pull/5738#discussion_r2002039912), i take it back